cloud/blockstore/libs/notify/notify_ut.cpp

```diff
@@ -128,4 +142,64 @@ Y_UNIT_TEST_SUITE(TNotifyTest)
     }
 }
 
+Y_UNIT_TEST_SUITE(TNotifyTestV2)
+{
+    Y_UNIT_TEST(ShouldNotifyDiskErrorV2)
+    {
+        auto service = CreateService(MakeConfigV2(), NCloud::NIamClient::CreateIamTokenClientStub());
+        service->Start();
+
+        auto r = service->Notify({
+            .CloudId = "yc-nbs",
+            .FolderId = "yc-nbs.folder",
+            .Timestamp = TInstant::ParseIso8601("2024-04-01T00:00:01Z"),
+            .Event = TDiskError{ .DiskId = "ShouldNotifyDiskErrorV2" },
+        }).GetValue(WaitTimeout);
+
+        UNIT_ASSERT_C(!HasError(r), r);
```

Comment:
In order not to duplicate the code, it is better to extract a test case to a function. Like this:
```cpp
auto CreateNotifyService(int version)
{
    return CreateService(
        version == 2
            ? MakeConfigV2()
            : MakeConfig(),
        NIamClient::CreateIamTokenClientStub());
}

void ShouldNotifyDiskErrorImpl(int version)
{
    auto service = CreateNotifyService(version);

    service->Start();

    auto r = service->Notify({
        .CloudId = "yc-nbs",
        .FolderId = "yc-nbs.folder",
        .Timestamp = TInstant::ParseIso8601("2024-04-01T00:00:01Z"),
        .Event = TDiskError{ .DiskId = "ShouldNotifyDiskErrorV" + ToString(version) },
    }).GetValue(WaitTimeout);

    UNIT_ASSERT_C(!HasError(r), r);

    service->Stop();
}

Y_UNIT_TEST(ShouldNotifyDiskError)
{
    ShouldNotifyDiskErrorImpl(1);
}

Y_UNIT_TEST(ShouldNotifyDiskErrorV2)
{
    ShouldNotifyDiskErrorImpl(2);
}
```